corehq/apps/dashboard/views.py

```diff
@@ -134,7 +135,12 @@ def can_edit_locations_not_users(request):
 
     can_view_commtrack_setup = lambda request: (request.project.commtrack_enabled)
 
-    can_view_exchange = lambda request: can_edit_apps(request) and not settings.ENTERPRISE_MODE
+    def can_view_exchange(request):
+        return (
+            can_edit_apps(request)
+            and not settings.ENTERPRISE_MODE
+            and not get_domain_master_link(request.domain)  # this isn't a linked domain
```

Comment:
Might be worth replacing `get_domain_master_link`  with something on the lines of `is_linked_domain` for better semantics.